crates/codegen/runtime/cargo/crate/src/runtime/parser/parser.rs.jinja2

```diff
@@ -64,6 +64,10 @@ impl Parser {
         }
     }
 
```

Comment:
WDYT of putting this in the `utils/LanguageFacts` type, given that it hosts all other version-related APIs?

The fact that we are using the parser under the hood is orthogonal.